frontend/src/composables/useDrawer.ts

```diff
@@ -38,7 +38,7 @@ type ModuleDrawerItem = {
  */
 type BakerDrawerItem = {
 	entityTypeName: 'baker'
-	id: number
+	hash: string
 }
```

Comment:
This is looks like a mistake, I would expect a baker to be associated with an id